src/benchmark.rs

```diff
@@ -0,0 +1,46 @@
+mod structures;
+mod gtfs_structures;
+mod algo;
+extern crate chrono;
+extern crate cpuprofiler;
+extern crate itertools;
+#[macro_use]
+extern crate serde_derive;
+use chrono::prelude::*;
+use cpuprofiler::PROFILER;
+use itertools::Itertools;
+
+fn main() {
+    let gtfs = gtfs_structures::Gtfs::new("./test_data/idf/").unwrap();
+    gtfs.print_stats();
+    let timetable = structures::Timetable::from_gtfs(gtfs, "2017-11-28", 1);
```

Comment:
taking this date by argument?

Reply:
I known a great lib for parsing command line argument https://github.com/TeXitoi/structopt 😇